docs/customization.md

```diff
@@ -174,8 +174,13 @@ This content appears at the bottom of every page's main content. More info for t
 
 Any HTML added to this file will be inserted before the closing `<head>` tag. This might include additional `<meta>`, `<link>`, or `<script>` tags.
 
-#### Example
-{: .no_toc }
+Note that by default, this file has the following contents:
+
+```html
+<link rel="shortcut icon" href="{{ 'favicon.ico' | relative_url }}" type="image/x-icon">
+```
+
+#### Example: Custom Favicon
 
 To add a custom favicon, create `_includes/head_custom.html` and add:
```

Comment:
Aren't users who are concerned about favicons, likely to want to add a whole bunch of links, rather than a single image?

Perhaps also worth mentioning that if one doesn't want a favicon, one needs to add an empty `head_custom.html`.

Reply:
Hm, I could definitely see a user just adding a custom favicon, since that's a relatively simple change (versus importing other script tags, etc. seems a bit more unlikely).

> Perhaps also worth mentioning that if one doesn't want a favicon, one needs to add an empty head_custom.html.

I can certainly note that!